### PR TITLE
update basic-features.md for v8.5

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,253 +20,253 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [DDL 算法断言](/sql-statements/sql-statement-alter-table.md) |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| 在单条语句中添加多列 |  Y  |  Y  |  Y  | Y | Y | E | E | E | E | E |
+| [更改列类型](/sql-statements/sql-statement-modify-column.md) |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [临时表](/temporary-tables.md) |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | N | N |
+| 并行 DDL |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N |
+| [添加索引加速](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N |
+| [元数据锁](/metadata-lock.md) |  Y  |  Y  | Y | Y | N | N | N | N | N |
+| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N |
+| [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL |  Y  |  Y  | Y | N | N | N | N | N | N | N |
+| [TiDB 加速建表](/accelerated-table-creation.md) | E | E | N | N | N | N | N | N | N | N |
```
Comment (Oreoxmt) [inline basic-features.md:140]:
```suggestion
| [TiDB 加速建表](/accelerated-table-creation.md) | Y | E | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 2]
```diff
@@ -20,253 +20,253 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Titan Level Merge](/storage-engine/titan-configuration.md#level-merge实验功能) | E |  E |  E  | E | E | E | E | E | E | E |
+| [使用 bucket 提高数据扫描并发度](/tune-region-performance.md#使用-bucket-增加并发) | E |  E |  E  | E | E | E | N | N | N | N |
+| [不可见索引](/sql-statements/sql-statement-create-index.md#不可见索引) | Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [复合主键](/constraints.md#主键约束) | Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [`CHECK` 约束](/constraints.md#check-约束) | Y |  Y |  Y  | N | N | N | N | N | N | N |
+| [唯一约束](/constraints.md#唯一约束) | Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [整型主键上的聚簇索引](/clustered-indexes.md) | Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [复合或非整型主键上的聚簇索引](/clustered-indexes.md) | Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | Y |  Y |  Y  | Y | N | N | N | N | N | N |
+| [外键约束](/constraints.md#外键约束) | E |  E |  E  | E | N | N | N | N | N | N |
```
Comment (lilin90) [inline basic-features.md:65]:
```suggestion
| [外键约束](/foreign-key.md) | Y |  E |  E  | E | N | N | N | N | N | N |
```

File: basic-features.md
Comment (qiancai) [inline basic-features.md:1]: preview: https://pr.pingcap-docsite-preview.pages.dev/zh/tidb/dev/basic-features

**Review by hfxsd — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: basic-features.md [hunk 3]
```diff
@@ -20,253 +20,253 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [后台任务资源管控](/tidb-resource-control.md#管理后台任务) | E | E | E | N | N | N | N | N | N | N |
+| [TiFlash 存算分离架构与 S3 支持](/tiflash/tiflash-disaggregated-and-s3.md) | Y | Y | Y | E | N | N | N | N | N | N |
+| [选择执行分布式执行框架任务的 TiDB 节点](/system-variables.md#tidb_service_scope-从-v740-版本开始引入) | Y | Y | Y | N | N | N | N | N | N | N |
+| 通过系统变量 [`tidb_enable_tso_follower_proxy`](/system-variables.md#tidb_enable_tso_follower_proxy-从-v530-版本开始引入) 控制 PD Follower Proxy 功能 | Y | Y | Y | Y | Y | Y | Y | Y | N | N |
+| 通过系统变量 [`pd_enable_follower_handle_region`](/system-variables.md#pd_enable_follower_handle_region-从-v760-版本开始引入) 控制 [Active PD Follower](/tune-region-performance.md#通过-active-pd-follower-提升-pd-region-信息查询服务的扩展能力) 功能 | Y | E | N | N | N | N | N | N | N | N |
+| [PD 微服务](/pd-microservices.md) | E | E | N | N | N | N | N | N | N | N |
+| [TiDB 分布式执行框架](/tidb-distributed-execution-framework.md) | Y | Y | Y | E | N | N | N | N | N | N |
+| [全局排序](/tidb-global-sort.md) | Y | Y | E | N | N | N | N | N | N | N |
+| [TiProxy](/tiproxy/tiproxy-overview.md) | Y | Y | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:270]:
```suggestion
| [Schema 缓存](/schema-cache.md) | Y | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 4]
```diff
@@ -20,254 +20,254 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Schema 缓存](/schema-cache.md) | Y | N | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:270]:
```suggestion
| [Schema 缓存](/schema-cache.md) | Y | N | N | N | N | N | N | N | N | N |

```

**Pull request merged**